hack/boilerplate-check.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+curDir=$(cd "$(dirname "$0")" && pwd)
+cd "${curDir}/../" || return
+
+check() {
+    # boilerplate check
+    echo "CHECK: boilerpalte, check code boilerplate"
```

Comment:
This sentence looks weird. Why not just use `Begin to check code boilerplate` here?

Reply:
Because I write `Begin to check code boilerplate` in Makefile. Maybe just remove this? WDYT?

Reply:
Yes, just keep one comment is OK.